README.md

```diff
@@ -287,6 +287,7 @@ var server = ParseServer({
     // 2. a callback function to be invoked to validate the password  
     validatorCallback: (password) => { return validatePassword(password) }, 
     doNotAllowUsername: true, // optional setting to disallow username in passwords
+    daysBeforeExpiry: 90, // threshold (days) for password expiry. Login fails if user does not reset the password within this period after signup/last reset. 
```

Comment:
@bhaskaryasa 
`daysBeforeExpiry` to me sounds like you are keeping a counter instead of it being a constant i.e., lets say I changed my password today. If this value is say: 90 days. `daysBeforeExpiry` name feels like if tomorrow the value be 89 days and keeps decrementing.

If you pls don't mind then I would really appreciate if u can change it to `maximumPasswordAge`